bindings/go/runtime/identity_match.go

```diff
@@ -56,6 +56,23 @@ func IdentityEqual(a Identity, b Identity) bool {
 	return a.Equal(b)
 }
 
+// IdentitySubset is a ChainableIdentityMatcher that checks if the identity sub is a subset of the identity base.
+// It is useful to check if an identity is a subset of another identity, and thus can be considered
+// an IdentityEqual matcher on a subset of the identity.
+//
+// Note that giving an empty subset will always return true, as an empty identity is a subset of any identity.
+func IdentitySubset(sub Identity, base Identity) bool {
+	if len(sub) > len(base) {
+		return false
+	}
+	for k, vsub := range sub {
+		if vm, found := base[k]; !found || vm != vsub {
+			return false
+		}
+	}
```

Comment:
I am having some trouble understanding this part.

`k` = key
`vsub` = value of subset?
`vm` = value of subset match?